### PR TITLE
Add candidate validation

### PR DESCRIPTION
**Pull request by pbochis**

### CODE REVIEW

File: app/scripts/util.js [hunk 1]
```diff
@@ -85,6 +103,7 @@ var util = {
 		}
 		page.redirect('/error');
 	},
+	emailRegex: '[a-z0-9!#$%&\'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&\'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?',
```
Comment (victorbalan) [inline app/scripts/util.js:106]: i think you can use polymer `paper-input`'s `allowedPattern` for this

Comment (pbochis) [inline app/scripts/util.js:106]: You're right, but I'd rather they do something wrong and then get a warning for it than just not being allowed to write stupid stuff and not understanding why. Remember, this form will be mostly used by non-technical people.

Comment (victorbalan) [inline app/scripts/util.js:106]: `pattern`, not `allowedPattern`.
Polymer docs - `A pattern to validate the input with.`

**Comment by lorenzleutgeb** [comment]
What's up with this PR?

File: app/elements/candidates/invite-candidate-input.html [hunk 2]
```diff
@@ -45,8 +50,8 @@
 				<p>{{challenges.0.Name}}</p>
 			</template>
 			<template is="dom-if" if="{{!isSingleChallenge}}">
-				<select id="challenges" on-change="_setChallenge">
-					<option selected disabled>Please select a challenge</option>
+				<select id="challenges" class$={{selectClass}} on-change="_setChallenge">
```
Comment (Exordian) [inline app/elements/candidates/invite-candidate-input.html:53]: https://elements.polymer-project.org/elements/paper-dropdown-menu

if we use this, we shoudn't need any custom validations and css-class hacks, and iz more beautiful

Comment (pbochis) [inline app/elements/candidates/invite-candidate-input.html:53]: When i coded this, the paper-dropdown was not available. Changing itn ow.

**Comment by pbochis** [comment]
#121 Fixes this.

**Pull request closed without merging**